testnet-contracts/contracts/ERC20.sol

```diff
@@ -0,0 +1,187 @@
+pragma solidity ^0.5.0;
+
+import "./IERC20.sol";
```

Comment:
does this file need to be here, rather than just imported from openzep?

Reply:
See https://github.com/swishlabsco/peggy/pull/2#discussion_r324489662

Reply:
@denalimarsh bump ^?

Reply:
https://github.com/swishlabsco/peggy/pull/2/commits/527279c23994a5a13546f05b2f87caedcd598c77